packages/modal/src/presenters/ModalPresenter.test.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import { generateId } from "@hig/utils";
+import ModalPresenter from "./ModalPresenter";
+
+describe("checkbox/presenters/ModalPresenter", () => {
+  afterEach(() => {
+    generateId.mockReset();
+  });
+
+  [
+    {
+      description: "renders without props",
+      props: {}
+    },
+    {
+      description: "renders with header children",
+      props: {
+        children: [<p key={"p"}>Body</p>],
+        headerChildren: [<h1 key={"h1"}>Title</h1>],
+        onCloseClick: function onCloseClick() {}
+      }
+    },
+    {
+      description: "renders with all props",
+      props: {
+        children: [<p key={"p"}>Body</p>],
```

Comment:
_nitpick:_ No need to escape strings in JSX.

Reply:
Updated in 5e9a1643